### ISSUE TITLE
Artipie server settings

### ISSUE THREAD

**Issue by g4s8** [artipie/artipie#14]
Parse Artipie server settings from yaml file as in white-paper:
https://github.com/artipie/white-paper/releases/download/30d9ae/white-paper.pdf

It's stored as a file with format:
```yaml
meta:
  storage:
    type: fs
    url: file:///artipie/settings
```

We need to construct `Storage` implementation from ASTO library using `storage` settings, where
`type` param may have value `fs` and `url` a link to root settings directory location.

I think it can be implemented as a class which implements `Storage` interface and encapsulates yaml configuration, e.g. `Storage settings = new AstoSettings(yaml)`.

**Comment by olegmoz** [comment]
After discussion decided to change setting format:
```
meta:
  storage:
    type: fs
    path: /artipie/storage
```
see https://github.com/artipie/white-paper/pull/9

**Comment by g4s8** [comment]
Fixed in #20

**Comment by 0crat** [comment]
Job `gh:artipie/artipie#14` is not assigned, can't get performer
